### PR TITLE
Update base image to latest LTS

### PR DESCRIPTION
**Pull request by TimoPtr**
I've update to the latest LTS available and also Gradle and Kotlin.
I had to remove the QT package since it has been removed, I don't know if the new version of the emulator needs it.

### CODE REVIEW

**Comment by thyrlian** [comment]
Thanks @TimoPtr for working on this PR, good initiative of upgrading the base image to the lastest LTS version.

The current Ubuntu version in use is 20.04.3 LTS, its end of standard support is April 2025 (according to Ubuntu's [releases page](https://wiki.ubuntu.com/Releases)).  The new one you proposed is 22.04 LTS, its end of standard support is April 2027.  In most cases, it's better to go with the latest LTS version.

I'll try my best to take some time reviewing your PR.  The catch is, the "smoke test" triggered by GitHub Actions only checks if all tools/commands are installed and can be found properly.  One concern is exactly as you mentioned - whether if the Android emulator can be launched flawlessly, due to the removal of the QT library.  I'll check it on my side, but please help check it as well on your side too, thank you.

File: android-sdk/Dockerfile [hunk 1]
```diff
@@ -18,14 +18,14 @@ ARG JDK_VERSION=11
 RUN dpkg --add-architecture i386 && \
     apt-get update && \
     apt-get dist-upgrade -y && \
-    apt-get install -y --no-install-recommends libncurses5:i386 libc6:i386 libstdc++6:i386 lib32gcc1 lib32ncurses6 lib32z1 zlib1g:i386 && \
+    apt-get install -y --no-install-recommends libncurses5:i386 libc6:i386 libstdc++6:i386 lib32gcc-s1 lib32ncurses6 lib32z1 zlib1g:i386 && \
     apt-get install -y --no-install-recommends openjdk-${JDK_VERSION}-jdk && \
-    apt-get install -y --no-install-recommends git wget unzip && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends qt5-default
+    apt-get install -y --no-install-recommends git wget curl unzip zip && \
```
[the remark this answers is not in the document]
Comment (thyrlian) [reply android-sdk/Dockerfile:23]: #CHANGE_REQUIRED

**Review by thyrlian — CHANGES_REQUESTED** [review]
Thank you @TimoPtr, I really appreciate your great work of upgrading the base image and keeping corresponding dependencies up-to-date.

I'm so excited that finally, we've proved that the emulator is working perfectly even without the system-level installation of the `qt` package.

I've tagged two places with #CHANGE_REQUIRED, please fix those two minor things, and then we are more than happy to merge the PR.  Thanks.

File: android-sdk/Dockerfile [hunk 2]
```diff
@@ -18,14 +18,14 @@ ARG JDK_VERSION=11
 RUN dpkg --add-architecture i386 && \
     apt-get update && \
     apt-get dist-upgrade -y && \
-    apt-get install -y --no-install-recommends libncurses5:i386 libc6:i386 libstdc++6:i386 lib32gcc1 lib32ncurses6 lib32z1 zlib1g:i386 && \
+    apt-get install -y --no-install-recommends libncurses5:i386 libc6:i386 libstdc++6:i386 lib32gcc-s1 lib32ncurses6 lib32z1 zlib1g:i386 && \
     apt-get install -y --no-install-recommends openjdk-${JDK_VERSION}-jdk && \
-    apt-get install -y --no-install-recommends git wget unzip && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends qt5-default
```
[the remark this answers is not in the document]
Comment (TimoPtr) [reply android-sdk/Dockerfile:24]: Thanks for your investigation 💪

**Comment by TimoPtr** [comment]
@thyrlian I've applied the change thanks for your review 👍

**Comment by thyrlian** [comment]
Great!  LGTM.  Thank you very much @TimoPtr.  I'll merge it once the CI workflow finishes.

**Review by thyrlian — APPROVED** [review]

**Comment by thyrlian** [comment]
Just published a new version [8.0](https://hub.docker.com/layers/android-sdk/thyrlian/android-sdk/8.0/images/sha256-a9c645fe6922a4b775d27e3738d544fab5526fcb3fc944fe58196aa645e755cb), thanks again @TimoPtr for your great work!

**Pull request merged**